app/models/container_node.rb

```diff
@@ -85,4 +86,20 @@ def evaluate_alert(_alert_id, _event)
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support external_logging_support'))
+    end
+  end
+
+  def external_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def external_logging_path
+    hostname = (ipaddress || name || "").split('.').first
+    query = "bool:(filter:(or:!((term:(hostname:#{hostname})),(term:(hostname:#{name})))))"
+    "/#/discover?_g=()&_a=(columns:!(_source),filters:!((meta:(disabled:!f,index:'.operations.*',key:hostname,negate:!f),#{query})),index:'.operations.*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'*')),sort:!(time,desc))"
```

Comment:
I am not sure that this URL is right, but it is working for me (I have added a gif, made it for the UI Patch but it fits here too).
To get this URL I opened Kibana and told it to filter by hostname. I then changed the query to catch the both fqdn and hostname and trimmed it a little. This is where the `_source` came from. To my understanding after playing with this a little this changes the columns that are displayed next to each message in the list. I will set this to the columns you suggested.
The field `level` is empty for me on all messages but I assume that future improvments with the data model will change that.